index.html

```diff
@@ -189,37 +189,40 @@ <h2 class="title">chess</h2>
             <div class="enclose">
                 <div class="base-card">
                     <div class="card-content">
-                        <h2 class="card-title">Music</h2>
+                        <h2 class="card-title">Melody Among us</h2>
+
                     <p class="card-body">
-                        Lorem ipsum dolor  elit. At earum, aspernatur id incidunt ducimus soluta optio voluptate. Dolor, asperiores porro.
+                        This saturday, NINAD club has been a wonderful releif in this stressful time of hustle-bustle for the grades D...
```

Comment:
This saturday, NINAD club has been a wonderful relief in this stressful time of hustle-bustle for the grades D...